.github/workflows/ci.yaml

```diff
@@ -217,35 +200,29 @@ jobs:
       # Ignore the failure of a step and avoid terminating the job.
       continue-on-error: true
       uses: actions/cache@v3
+      id: deps_cache
       with:
         # This path is specific to Ubuntu
         path: ${{ github.workspace }}/cget
         # Look to see if there is a cache hit for the corresponding requirements file
         key: 
           ${{ matrix.os }}-cget-4-${{ hashFiles('requirements.txt', 'dev-requirements.txt') }}
+        restore-keys: |
```

Comment:
Why is that ? 

It would try to first restore cache by using primary key that is key defined under field `key:` 

If that doesn't exist then, it would try to use `restore-keys`.   I don't see why we need to duplicate keys in both `key:` and `restore-keys:`.

Reply:
OK I see, I didnt know it would restore the `key` first before trying the `restore-keys`.